Orochi/OrochiUtils.h

```diff
@@ -67,6 +70,6 @@ class OrochiUtils
 	}
 
 public:
-	static char* s_cacheDirectory;
-	static std::map<std::string, oroFunction> s_kernelMap;
+	std::string m_cacheDirectory;
+	std::map<std::string, oroFunction> m_kernelMap;
```

Comment:
```suggestion
	std::unordered_map<std::string, oroFunction> m_kernelMap;
```
[Minor] Suggestion: use the unordered map for slightly better performance.